mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1279,6 +1280,9 @@ def _generate_model_predictions(self):
 
             import tiktoken
 
+            # ref: https://github.com/openai/tiktoken/issues/75
+            os.environ["TIKTOKEN_CACHE_DIR"] = ""
+
             encoding = tiktoken.get_encoding("cl100k_base")
```

Comment:
Can we move this logic inside of `compute_num_tokens` function? So its only used when users compute_num_tokens and its not default?

Reply:
I think we need to set this before we download the encoding which happens here on L1286